server/main.js

```diff
@@ -1,49 +1,48 @@
 import { Meteor } from 'meteor/meteor';
 
-import { Players } from '../imports/api/Players.js';
-import { HongKongHands } from '../imports/api/GameDatabases.js'
+import Players from '../imports/api/Players';
+import { HongKongHands } from '../imports/api/GameDatabases'
```

Comment:
`GameDatabases` actually has both `HongKongHands` and `JapaneseHands`. If we were to separate them out into files, we could do this. However, we have to import them by name for now.